extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -644,7 +645,9 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
           if (rtcResponse.response[key]) {
             this.jsonTargeting_[key] =
                 !!this.jsonTargeting_[key] ?
-                deepMerge(this.jsonTargeting_[key], rtcResponse.response[key]) :
+                deepMerge(this.jsonTargeting_[key],
+                    this.rewriteRtcKeys_(rtcResponse.response[key],
+                        rtcResponse.callout)) :
                 rtcResponse.response[key];
```

Comment:
Don't you need to rewrite the keys here as well?  This should be:
if (rtcResponse.response[key]) {
   const rewrittenKeys = this.rewriteRtcKeys_(rtcResponse.response[key], rtcResponse.callout);
   this.jsonTargeting_[key] = !!this.jsonTargeting_[key] ? deepMerge(this.jsonTargeting_[key], rewrittenKeys) : rewrittenKeys);
}

Reply:
done